src/ConnectedMode.UnitTests/Suppressions/TimedUpdateHandlerTests.cs

```diff
@@ -112,7 +138,8 @@ private static ITimerFactory CreateTimerFactory(ITimer timer)
             return timerFactory.Object;
         }
 
-        private static TimedUpdateHandler CreateTestSubject(ISuppressionIssueStoreUpdater suppressionIssueStoreUpdater = null,
+        private static TimedUpdateHandler CreateTestSubject(IActiveSolutionBoundTracker activeSolutionBoundTracker, 
```

Comment:
I was not sure if we should allow activeSolutionBoundTracker to be null here, personally, I am against it because I think neither standalone nor connected mode should be the default test state and that each test should define which state it is in.

Reply:
From the point of view of the tests we have to supply a non-null value, otherwise we'll get a null-ref exception in the constructor when it tries to register the event handler.